displayLinac.py

```diff
@@ -132,13 +143,15 @@ def create_faults(self):
             tlc = csvFaultDict["Three Letter Code"]
             ok_condition = csvFaultDict["OK If Equal To"]
             fault_condition = csvFaultDict["Faulted If Equal To"]
+            csv_prefix = csvFaultDict["PV Prefix"]
```

Comment:
Is this used anywhere that isn't in the hash function?

Reply:
It should only be being used for the hash function specifically only to differentiate the xcor, ycor, and quad MGT fault

Reply:
Ohhhh because they're the same otherwise. That's so annoying lol Good job catching that!